edge-agent/docker/linux/arm64v8/base/Dockerfile

```diff
@@ -1,7 +1,8 @@
-ARG base_tag=3.0-runtime-bionic-arm64v8
+ARG base_tag=3.0-bionic-arm64v8
 ARG num_procs=4
 
-FROM microsoft/dotnet:${base_tag}
+FROM mcr.microsoft.com/dotnet/core/runtime:${base_tag}
```

Comment:
are you going to update base image for Linux and Windows x64 in another PR?

Reply:
Nops. That Anca will do on her PR. This was just for ARM since I needed to update the base image.